zigbee2mqtt/DOCS.md

```diff
@@ -0,0 +1,63 @@
+# Home Assistant Add-on: Zigbee2mqtt
+
+# Pairing
+
+The suggested way to pair your devices is to enable zigbee2mqtt's `permit_join` option from within Home Assistant using MQTT rather than through the add-on's user interface.
+
+You can use [Zigbee2MqttAssistant](https://github.com/yllibed/Zigbee2MqttAssistant) if you don't want to use MQTT directly.
```

Comment:
Next Zigbee2MQTT version (1 oct release) will contain a built-in frontend: https://github.com/Koenkk/zigbee2mqtt/issues/4266

Reply:
Nice. I actually started a new lightweight addon to avoid a full .net one for such tasks. Good thing that I can drop that :-)

I think we should release this before next version of z2m and move to dev branch workflow for next one. 

I will update docs when new UI is available. I hope it will work with "ingress" for add-ons

Reply:
@Koenkk created https://github.com/danielwelch/hassio-zigbee2mqtt/issues/397 to track the update